src/core/services/gesture/gesture.js

```diff
@@ -361,10 +361,11 @@ angular.module('material.core')
     bubbles: true,
     cancelable: true
   };
+
   /*
    * NOTE: dispatchEvent is very performance sensitive. 
```

Comment:
The name here probably needs to be changed as well
OR 
I'm not sure that the name needs to specifically contain jqLite or if it would be enough to just add that to the comment here.